### ISSUE TITLE
Support for OpenELEC 4.0

### ISSUE THREAD

**Issue by timhansinger** [tvdzwan/hyperion#101]
I wanted to ask if support for OpenELEC 4.0 is planned? Currently it's not working. Hyperion-Service starts, but stops running after a few seconds while playing a movie.

**Comment by NicoHood** [comment]
Ahm just a question regarding this:
For what is the XBMC checker used? I am currently running Kodi + Hyperion on my linux laptop and I alwats get XBMC connection errors. It seems that I dont need this interface. The protoserver + Kodi addin just works fine. So where do I need this XBMC checker? Just on the raspi?

**Comment by Gamadril** [comment]
xbmc checker uses Kodi's RPC API to get the current state of Kodi, like screensaver, menu, movie playing etc. Since there is also a build for x86 with native framegrabber, you need the xmbc checker to get Kodi's state. Not sure about protoserver - never used it...

**Comment by NicoHood** [comment]
Oh I think I did not compile the framegrabber or it just doesnt work. So With the proto server I dont need this? Which one is the better solution? I only need ambilight inside the video, thats totally okay. I just need a fast and stable version running with the best picture (led) quality.

**Comment by Gamadril** [comment]
native grabber will be always faster and (probably less cpu consuming - not tested however). test the x86 build here: https://github.com/tvdzwan/hyperion/issues/62#issuecomment-103026587

**Comment by NicoHood** [comment]
Why dont you open a PR for this? Nativ linux build would be very nice. I have no problem with self recompiling, however this would be a good enhancement.

**Comment by Gamadril** [comment]
I did some modifications of the install script and haven't tested if it breaks any other platforms... the native grabber works only with the framebuffer (/dev/fb), so it will not work with an X server. There is an X grabber avaialable running as a separate process, but it also uses protobuffer for communication with hyperion, so probably not a real improvement to your current solution. I worked on a separate hyperion branch which does not depend on QT and integrates the X grabber natively, however I have no time to test it propperly before making it public.
